### ISSUE TITLE
Error: EPERM, unlink with bower install in msysgit

### ISSUE THREAD

**Issue by dotnetCarpenter** [bower/bower#1692]
I'm trying to install a fork of moment.js from my repo on bitbucket using [msysgit](https://msysgit.github.io/) on win7. I ran `Git Bash` as administrator and got the same error. I had the *normal* moment.js installed before in the same project,un-installed moment.js and cleared bower's cache with `bower cache clean`. This is the first time I've had this issue. Is it because of the [pre-release version](http://semver.org/)?

Error details:
```
bower resolved      git@bitbucket.org:indecks/moment.git#2.9.0-en.gb.1
bower install       moment#2.9.0-en.gb.1
bower EPERM         EPERM, unlink 'c:\Users\dotnet\Projects\weaddmotion\armadillo\indecks-component-datepicker\bower_components\moment'

Stack trace:
Error: EPERM, unlink 'c:\Users\dotnet\Projects\weaddmotion\armadillo\indecks-component-datepicker\bower_components\moment'
    at Error (native)

Console trace:
Trace
    at StandardRenderer.error (c:\Users\dotnet\AppData\Roaming\npm\node_modules\bower\lib\renderers\StandardRenderer.js:82:17)
    at Logger.<anonymous> (c:\Users\dotnet\AppData\Roaming\npm\node_modules\bower\bin\bower:110:22)
    at Logger.emit (events.js:107:17)
    at Logger.emit (c:\Users\dotnet\AppData\Roaming\npm\node_modules\bower\node_modules\bower-logger\lib\Logger.js:29:39)
    at c:\Users\dotnet\AppData\Roaming\npm\node_modules\bower\lib\commands\index.js:40:20
    at _rejected (c:\Users\dotnet\AppData\Roaming\npm\node_modules\bower\node_modules\q\q.js:797:24)
    at c:\Users\dotnet\AppData\Roaming\npm\node_modules\bower\node_modules\q\q.js:823:30
    at Promise.when (c:\Users\dotnet\AppData\Roaming\npm\node_modules\bower\node_modules\q\q.js:1035:31)
    at Promise.promise.promiseDispatch (c:\Users\dotnet\AppData\Roaming\npm\node_modules\bower\node_modules\q\q.js:741:41)
    at c:\Users\dotnet\AppData\Roaming\npm\node_modules\bower\node_modules\q\q.js:557:44

System info:
Bower version: 1.3.12
Node version: 0.12.0
OS: Windows_NT 6.1.7601 x64
```

**Comment by dotnetCarpenter** [comment]
Hmm.. It seems to be related to [atom](https://atom.io/). I tried to manually delete the old moment folder from msysgit: `rm -rf moment` but was denied. I then tried to open the folder in explorer but got another denied alert. After checking the properties, I found that the Read-only flag was set but I was denied to unset it. After closing atom, in which I had the project open in, the folder disappeared. I then did another `bower i git@bitbucket.org:indecks/moment.git#2.9.0-en.gb.1 --save` and this time it worked.
Curious...

Feel free to close this issue if no one can replicate.
